src/navigation/navigationOptions.js

```diff
@@ -1,20 +1,11 @@
 // @flow
-
-import { Platform } from "react-native";
-import colors from "styles/tailwindColors";
+import ContextHeader from "./ContextHeader";
 
 const showHeader: Object = {
   headerShown: true,
   headerBackTitleVisible: false,
   headerShadowVisible: false,
-  headerTintColor: colors.black,
-  // Note: left header is not supported on iOS
-  // so we would need to build a custom header for this:
-  // https://reactnavigation.org/docs/native-stack-navigator#headertitlealign
-  headerTitleStyle: {
-    fontSize: 24,
-    fontFamily: Platform.OS === "ios" ? "Whitney-Medium" : "Whitney-Medium-Pro"
-  }
+  header: ContextHeader
```

Comment:
I can make that change, but does it seem reasonable to have this component just be more flexible? That way components that do have a header will maintain the same styles throughout the app?

Reply:
My two cents is that there aren't many designs in Figma that use this type of header, so this feels like the special case rather than the default header. I think the default header should have center-aligned text, no shadow, support for right buttons, and no stickiness over a scrollview -- aka the header we already have in the app.

And it would be ideal if we can make that default header flexible to accept a shadow, subtitle, stickiness, and left-aligned text, but if react-navigation doesn't support that easily I think it's fine to keep it separate.

Reply:
This seems like a reasonable approach to me.